src/web/src/app/space/invitation-list/component.ts

```diff
@@ -87,8 +87,8 @@ export class SpaceInvitationListComponent implements OnInit {
 			distinctUntilChanged(), // Optional: ensure distinct values before making requests
 			switchMap(value =>  this.auth.getList(10, 0, value).pipe(
 				map(res => {
-					this.hasUsers = res.length > 0;
-					return res;
+					this.hasUsers = res["list"].length > 0;
+					return res["list"];
```

Comment:
Тут кажется тебе вернётся уже описанный тип у которого есть такое поле, так что можно его использовать напрямую.
```suggestion
					this.hasUsers = res.list.length > 0;
					return res.list;
```